README.md

```diff
@@ -1,6 +1,16 @@
 # DevOps Apprenticeship: Project Exercise
 
-## Setup
+## With docker 
+### Setup 
+run `docker compose run generate-env` to generate a .env file.
```

Comment:
Today I learned that `docker compose` exists and is subtly different to `docker-compose`!

Despite that, this didn't work for me out of the box sadly; the issue I had was the `docker compose` (exactly as with `docker-compose`) would complain that the .env file doesn't exist for the other services, even though it's not trying to run them yet. Once it's been forced to run once, it'll work correctly though